src/seedu/addressbook/parser/Parser.java

```diff
@@ -226,5 +228,119 @@ private Command prepareFind(String args) {
         return new FindCommand(keywordSet);
     }
 
+    /**
+     * Feedback a suggested command to the user.
+     *
+     * @param incorrectCommandWord the incorrect command word
+     * @return the suggested command
+     */
+    private IncorrectCommand suggestCommand(String incorrectCommandWord) {
```

Comment:
You are returning a `command`, so your method name should reflect that. `suggestCommand` is an action. `suggestedCommand` is a more appropriate name